packages/notification/channel/mailgun.ts

```diff
@@ -22,17 +22,24 @@
  * SOFTWARE.                                                                      *
  **********************************************************************************/
 
-import Mailgun from 'mailgun.js'
-import formData from 'form-data'
+import FormData from 'form-data'
 import Joi from 'joi'
 import type { NotificationMessage } from '.'
 import Mailgen from 'mailgen'
+import { sendHttpRequest } from '../utils/http'
 
-type NotificationData = {
+export type NotificationData = {
```

Comment:
Could you elaborate why it needs to be exported?

Reply:
same with other, need to be exported to be used in neosense